source/Halibut.Tests/Support/TestAttributes/LatestClientAndPreviousServiceVersionsTestCasesAttribute.cs

```diff
@@ -14,18 +15,19 @@ public class LatestClientAndPreviousServiceVersionsTestCasesAttribute : TestCase
             bool testNetworkConditions = true,
             bool testListening = true,
             bool testPolling = true,
-            bool testAsyncAndSyncClients = true
+            bool testAsyncAndSyncClients = true,
+            bool testAsyncServicesAsWell = false // False means only the sync service will be tested.
```

Comment:
In `LatestClientAndPreviousServiceVersionBuilder`, we state that "The service is external and so can not be made to use async".

So therefore, should we have `testAsyncServicesAsWell` on this test case attribute? (as it is also 'previous service')

Reply:
You can change this if it turns out to be important.